cypress.config.ts

```diff
@@ -11,6 +11,7 @@ export default defineConfig({
       hideCredentials: true,
       TEST_USERNAME: '***',
       TEST_PASSWORD: '***',
+      TEST_GOOGLE_PASSWORD: '***',
```

Comment:
why do we require this ?

Reply:
It's not required for this PR. I'll take it off.